curses.c

```diff
@@ -77,15 +77,27 @@ int main(void)
     char tmp_board[ARRAY_SIZE][ARRAY_SIZE];
     // int sleep_time = 1000;
     int character_code;
-    int location_x = 5;
-    int location_y = 5;
+    int location_x = 5, location_y = 5, player_big = 0;
```

Comment:
I like that we have mush_x and mush_y. Do you think this is a good time to change location_x and location_y to just player_x and player_y?

Reply:
done